src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/TextDifferencing/SourceTextDifferBenchmark.cs

```diff
@@ -19,7 +21,10 @@ public class SourceTextDifferBenchmark
     [GlobalSetup]
     public void GlobalSetup()
     {
-        var largeFileText = Resources.GetResourceText("MSN.cshtml");
+        var msnCshtmlDiskPath = Path.Combine(RepoRoot, "src", "Razor", "test", "testapps", "ComponentApp", "Components", "Pages", "MSN.cshtml");
+        using var fileStream = new FileStream(msnCshtmlDiskPath, FileMode.Open);
+        var reader = new StreamReader(fileStream);
+        var largeFileText = reader.ReadToEnd();
```

Comment:
Why are we switching back to reading this from disk?

Reply:
Maybe this is an unintended change included in the PR? This benchmark doesn't seem to be related.

Reply:
Oh, I see why this is needed now. MSN.cshtml is moving out of the benchmark resources for some reason, though it's not clear why.

Reply:
I'm happy to move in that direction and have added overloads to RazorLanguageServerBenchmarkBase in order to make it easier to transition.
I did notice while doing this, however, that after restoring the SourceTextDifferBenchmark but not moving the content, there were no compile errors but there still were the predicted runtime errors.